index.html

```diff
@@ -4,9 +4,152 @@
+		<nav class="navbar hide" id="mobile-menu">
+		  <div>
+        <img src="assets/images/icons/Cancel-Icon.svg" alt="menu icon" class="menu-icon" id="cancle-icon">
+			</div>
+		  <ul class="nav-items">
+				<li class="nav-item"><a href="about.html">About</a></li>
+				<li class="nav-item"><a href="">Program</a></li>
+				<li class="nav-item"><a href="">Join</a></li>
+				<li class="nav-item"><a href="">Sponser</a></li>
+				<li class="nav-item"><a href="">News</a></li>
+				<li class="nav-item nav-btn"><a href="">CC Campaign</a></li>
+			</ul>
+		</nav>
+	</header>
+	<header class="desktop">
+    <article class="toolbar">
+      <a href="#"><img src="assets/images/icons/facebook-icon.png" alt="facebook icon" class="toolbar-item"></a>
+			<a href="#"><img src="assets/images/icons/twitter.png" alt="twitter icon" class="toolbar-item"></a>
+			<a href="#" class="toolbar-item">English</a>
+			<a href="#" class="toolbar-item">My page</a>
+			<a href="#" class="toolbar-item">Logout</a>
+		</article>
+    <nav class="navbar-desktop">
+			<a href="index.html"><img src="assets/images/logo.png" alt="logo" class="nav-logo"></a>
+			<ul class="nav-items-desktop">
+        <li><a href="about.html"class="nav-item-desktop">About</a></li>
+				<li><a href="#" class="nav-item-desktop">Program</a></li>
+				<li><a href="#" class="nav-item-desktop">Join</a></li>
+				<li><a href="#" class="nav-item-desktop">Sponser</a></li>
+				<li><a href="#" class="nav-item-desktop">News</a></li>
+				<li class="nav-item-desktop nav-btn">
+           <button class="top-btn">JS Hackathon</button>
+				</li>
 
+			</ul>
+		</nav>
+	</header>
+  <div class="container show" id="main-body">
+		<header class="mobile">
+			<nav class="navbar">
+				<div>
+					<img src="assets/images/icons/hamburger-icon.svg" alt="menu icon" class="menu-icon" id="menu-icon"></div>
+			</nav>
+		</header>
+		<section class="hero">
+		  <div class="hero-heading">
+				<span><p class="supporting-text">"Hello coding world"</p></span>
+				<h1 class="primary-text">Javascript Devs Hackathon 2023</h1>
+			</div>
+			<div>
+				<p class="description-hero">
+					Javascript Hackathon is an annual weekend long beginer friendly hackathon series for Javascript enthusiasts with developers from all over the world and different skill levels.
+				</p>
+			</div>
+			<div class="date-venue">
+				<h6 class="date">2023.02.04(SAT) ~ 5(SUN)</h6>
+				<p class="description venue">@ National Museum of Karea, Art Center Nabi and more
+        </p>
+			</div>
+		</section>
+		<section class="program">
+			<h2 class="main-title">Main Program</h2>
+			<hl class="divider"></hl>
+			<div class="program-cards-container">
+				<div class="program-card">
+					<img src="assets/images/icons/program-icon.png" alt="Program icon" class="program-icon">
+					<h3 class="program-title">Lecture</h3>
+					<p class="program-description">Listen to the speakers from various countries about the messages of sharing and opening.</p>
```

Comment:
- [Optional] Good job making the requested changes @FevenSeyfu, Just to make your design better make sure your text s and other contents always stay responsive within the card or container it is contained in. 

### This is a screenshot taken from a 770px screen size. 
![Screenshot 2023-01-14 141520](https://user-images.githubusercontent.com/69077061/212469460-1d1f2bcf-cfb0-4d73-8111-10b4ba347bfc.png)
![Screenshot 2023-01-14 141721](https://user-images.githubusercontent.com/69077061/212469463-5d25a72e-9dae-43f2-8a5a-ff4e243d1a9f.png)
![Screenshot 2023-01-14 142012](https://user-images.githubusercontent.com/69077061/212469464-2b0b0b2a-e160-4161-b75e-cf97d584238b.png)

> Remember in the future hiring manager and people, in general, are going to visit your projects on different screens.

Reply:
Fixed padding issues for nav bar and main program section.